CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to JupyterLab
```

Comment:
There are a few tools in here that we don't have set up yet in this repo (testing, prettier, lint-staged documentation). Could you either add those to this PR or remove the references to them from this document?

Reply:
Ok, I removed referances to all tools that are not set up yet. I will add those sections back as they are added.

Reply:
Thanks!